1-org/envs/shared/projects.tf

```diff
@@ -94,11 +103,15 @@ module "interconnect" {
   billing_account             = var.billing_account
   folder_id                   = google_folder.common.id
   skip_gcloud_download        = var.skip_gcloud_download
+  activate_apis               = ["billingbudgets.googleapis.com"]
```

Comment:
This change is replacing the default `activate_apis` list `["compute.googleapis.com"]` with `["billingbudgets.googleapis.com"]`. 

To have no side effects we could use `["billingbudgets.googleapis.com", "compute.googleapis.com"]`

Reply:
done